### ISSUE TITLE
Prevalence update not run since 2022-01-13?

### ISSUE THREAD

**Issue by Joegratz** [microCOVID/microCOVID#1323]
**Describe the bug**
Until 2022-01-13, prevalence updates were running daily. No prevalence update has happened since then. 

**Link to microCOVID scenario**
https://github.com/microCOVID/microCOVID/pulls?q=label%3Aauto-prevalence-update+

**To Reproduce**
1. Look at last updated date
2. Note that it is more than a week ago

**Expected behavior**
Daily prevalence updates.

**Is this a regression?**
Yes. 

**Additional context / screenshots**
Add any other context about the problem here, including any screenshots.

**Comment by zestyping** [comment]
Hello, I was visiting to report this problem as well.  The last data I see on microcovid.org is from January 12.

![image](https://user-images.githubusercontent.com/236086/150718265-bb6943a9-dff8-4d8e-9cab-652519b86a5e.png)

I made covidmeter.org as an easy way to do faster estimates based on microcovid.org's model.  We fetch data by pulling the `update_prevalence` script from the microCOVID repo and running it, like so:

https://github.com/zestyping/covidmeter/blob/main/get_prevalence_js

It stopped working on January 13 because Antarctica has zero population.  So I simply patched the code to skip Antarctica, and it runs just fine now:

https://github.com/zestyping/covidmeter/commit/611190cd07c50d84b22eeae0d9d30de4ba87009a

You might want to do something similar to get microcovid.org working again.  Thanks for making this project!
